### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by som23ya** [fbdevelopercircles/FbDevcCommunityContent#94]
Blogs, articles and videos in the readme file should be sorted according to projects and apps.
Currently they look jumbled up and difficult to navigate.

**Comment by jordanliu** [comment]
I agree, it's pretty hard to digest what's happening - it can be a bit overwhelming

**Comment by spyke01** [comment]
I'll take a swing at this today and submit a Pull Request for review.
